custom/ewsghana/reports/email_reports.py

```diff
@@ -61,7 +61,7 @@ def _stock_status(status, loc):
                                       'stockout': 0, 'low': 0, 'overstock': 0, 'adequate': 0}
 
         for location in locations:
-            location_products = list(location.products)
+            location_products = filter(lambda product: not product.is_archived, location.products)
```

Comment:
`products` uses a django queryset - I think that the implementation of `products` should have `.filter(is_archived=False)` in it instead.